### ISSUE TITLE
Documentation and user guides

### ISSUE THREAD

**Issue by provanguard** [ledoge/novideo_srgb#11]
Would it be possible for you to write up some step-by-step documentation about how to use the application, and in particular including the following aspects:
- Calibration with DisplayCAL and using the profile with the novideo_srgb tool
- Calibration with DisplayCAL to gamma 2.2 vs to sRGB and using the novideo_srgb tool
- Calibration in user mode vs sRGB mode (with locked monitor settings) on a wide color gamut monitor and using the novideo_srgb tool (optional)

In general I would like there to be an easy guide that I could always refer to, to perform full calibration of my ASUS PG329Q wide color gamut monitor using DisplayCAL and then use the novideo_srgb tool to ensure the image on the screen is clamped to sRGB color space. I normally calibrate to gamma 2.2.

**Comment by ledoge** [comment]
To be honest, I'm not entirely sure myself as to what options you'd ideally use in DisplayCAL to achieve the best possible color accuracy with this tool. However, you can think of it like doing the work that color management would normally do in some program, which e.g. means that it shouldn't really matter which gamma you actually calibrate to.

If you don't care about the exact black point e.g. for accurate BT.1886 tracking, you should be perfectly fine by just calibrating with DisplayCAL's default sRGB settings, optionally with a different white point and/or gamma target (or setting gamma to "as measured" should also be fine). Then just load the ICC profile into novideo_srgb, enable gamma calibration and select sRGB/2.2 gamma there, and that's what your screen will end up being calibrated to (feel free to verify this with DisplayCAL's verification tab, and make sure that "Use simulation profile as display profile" is checked).

If you do care about the exact black point, I believe you should change "Profile type" to "Curves + matrix" and make sure that "Black point compensation" is disabled.

I think in theory enabling the sRGB mode in your monitor shouldn't result in any worse results as long as it doesn't reduce the sRGB coverage (which is a non-trivial assumption), and I don't see how it could give you any better results. So I would definitely leave it off.

**Comment by provanguard** [comment]
Here are my settings in DisplayCAL. I am not an expert in this app, nor in calibration in general. Just tried to replicate the settings that xrite i1profiler is using as recommended for photo editing. 

![capture_20211228_200214](https://user-images.githubusercontent.com/19497436/147598840-43f72ef2-a299-42fd-97cb-461d72340ca9.jpg)

![capture_20211228_200222](https://user-images.githubusercontent.com/19497436/147598862-296c093b-3255-4d63-b8bd-3da55a943d96.jpg)

![capture_20211228_200226](https://user-images.githubusercontent.com/19497436/147598879-36a26825-0cdc-4b82-a8b1-221bcd50d5a6.jpg)

**Comment by provanguard** [comment]
What should I start with? DisplayCAL calibration on an unclamped monitor? Or on already clamped?
Should I load the profile generated by DisplayCAL into the novideo_srgb tool and enable clamp? Is that it? What options should I chose in the tool?

I am still running the version 0.3-test where I modified the RGB points, based on what I found in the DisplayCAL profile information.

![capture_20211228_201247](https://user-images.githubusercontent.com/19497436/147599479-0697167b-d50d-4e94-9523-8cdc0ee34633.jpg)

![capture_20211228_201845](https://user-images.githubusercontent.com/19497436/147599926-49e97720-6a61-4080-afaf-82482e780e7e.jpg)

When I am now testing the same profile loaded into the tool with selected gamma 2.2, I have an impression (not measured yet) that in the new version I am getting more saturation. So my conclusion is that I am using the tool in the wrong way and hence I asked for the more detailed guide.

By the way, I am using a youtube video containing different faces, which I found very easy to quickly check if there is too much saturation of red colors: https://youtu.be/BQQxNa6U6X4?t=256

**Comment by provanguard** [comment]
Please note that values shown in Use EDID primaries setting are quite different from the ones provided by DisplayCAL. Also, there are 3 decimal places in the tool, while DisplayCAL has 4.
I am not sure what that means, but I assume that this is the reason to have the option to load the ICM profile created by DisplayCAL and read the values from there instead of the EDID.

**Comment by provanguard** [comment]
Here are the profiles I generated using DisplayCAL, back when first testing version 0.3-test - one before using the clamp and one after. sRGB coverage is at 0,995 for the first and 0,9945 for the second.

[profiles.zip](https://github.com/ledoge/novideo_srgb/files/7786153/profiles.zip)

**Comment by ledoge** [comment]
>What should I start with? DisplayCAL calibration on an unclamped monitor? 

Yes.

>Please note that values shown in Use EDID primaries setting are quite different from the ones provided by DisplayCAL. Also, there are 3 decimal places in the tool, while DisplayCAL has 4.

That and the additional saturation you're seeing now is due to the EDID values being inaccurate. The highlighted values in that DisplayCAL screenshot are the actual measured ones, which are accurate (assuming your colorimeter is not broken). These are also (effectively) being used by novideo_srgb now with the ICC profile selected, along with the VCGT and TRC curves.

Your usage of DisplayCAL and novideo_srgb looks correct from those screenshots – run DisplayCAL on the unclamped display, then load the resulting profile exactly like in your novideo_srgb screenshot, enable the clamp and you should be good to go. If you want to use sRGB gamma instead now there's no need to re-calibrate, just change the gamma in novideo_srgb to sRGB.

**Comment by aufkrawall** [comment]
Yep. The way you recommended to test (but still XYZ + matrix profile type), I can confirm that I could nicely bring down delta values from terrible (display's default rec709 oversaturation) to okayish (clamping with default EDID chromaticity values) to good (clamping with ICC chromaticity values) to great (clamping with ICC chromaticity values + corrected gamma 2.2). Simply fantastic. :)

**Comment by provanguard** [comment]
Just loaded the latest version 1.2. What is the difference between absolute and relative gamma? 
Why are the new default values for relative and absolute gamma at 2.2000000476837158 ?

**Comment by ledoge** [comment]
>Why are the new default values for relative and absolute gamma at 2.2000000476837158 ?

Oops, that's a bug, thanks for catching it. Fixed it now.

>What is the difference between absolute and relative gamma?

Absolute gamma means that the exponent used is the actual value you entered, while relative calculates a new exponent so that the grayscale level at 50% input level is equivalent to what it would be on a display with a black level of zero and the exponent you entered. If your ICC profile includes black point compensation (or your display actually has infinite contrast), there will be no difference between them.

**Comment by provanguard** [comment]
Given I have a LED monitor, which does not have an infinite contrast, which setting is recommended, relative or absolute?

**Comment by ledoge** [comment]
Purely personal preference. I think DisplayCAL uses relative by default (I believe you need to enable advanced options or whatever it's called to see it?). Not sure if that implementation works exactly the same as mine, but I based it off an explanation of how it works that I read at some point.

**Comment by provanguard** [comment]
Perfect :) I think this new addition of using the profile and setting the gamma is a great killer option, which makes it much easier to use the app after calibration. Thanks!

A minor request, since I usually pin the app to the start menu, could you add an icon to the executable?
And out of curiosity, what is the meaning of the app name ? :)

**Comment by ledoge** [comment]
I would definitely add an icon if I had one, but someone else would have to make it as I'm definitely not artistically inclined.

Novideo is a bastardization of NVIDIA used on /r/AyyMD, I used it because I'm pretty sure naming anything after a company or a trademarked name is a bad idea (and I think it's funny).
